app/helpers/publishing_helper.rb

```diff
@@ -0,0 +1,11 @@
+module PublishingHelper
+  def remote_publish!(&block)
```

Comment:
I agree, although I'm kinda struggling to find something better... maybe something like `remote_write!` but again that feels very generic.